clients/githubrepo/graphql.go

```diff
@@ -224,8 +243,12 @@ func commitsFrom(data *graphqlData, repoOwner, repoName string) ([]clients.Commi
 				})
 			}
 			for _, review := range pr.Reviews.Nodes {
+				author := clients.User{
+					Login: string(review.Author.Login),
+				}
 				associatedPR.Reviews = append(associatedPR.Reviews, clients.Review{
-					State: string(review.State),
+					State:  string(review.State),
+					Author: &author,
```

Comment:
this part was missing so reviewers' names were not populated. PTAL that was not intentional or if it may breaks something.

Reply:
LGTM, thanks for catching this!